src/containers/ProductDetails/ProductDetailsView.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Screen from '../../components/Screen'
+import Button from '../../components/Button'
+import { Link } from 'react-router-dom'
+
+const ProductDetailsView = (props) => {
```

Comment:
The same here with `Object destructuring` it get more compact and in quick look you can figure out props used 
```jsx
const ProductDetailsView = ({product, addToCart}) => (
  <Screen>
```